tasks/dist/css

```diff
@@ -0,0 +1 @@
+node-sass --output-style expanded --source-map true --precision 6 docs/assets/scss/docs.scss docs/assets/css/docs.css
```

Comment:
probably better if you reference the local npm installs of these (in `.bin` ) so that people don't need global installs 👍